docs/gettingStarted/quickStart.rst

```diff
@@ -185,7 +185,12 @@ Running the example
    back together---is compartmentalized into a **job**. Each job can specify its own resource requirements and will
    only be run after the jobs it depends upon have run. Jobs without dependencies will be run in parallel.
 
-#. Run with custom options::
+.. note::
+        Delete ``fileToSort.txt`` before moving on to #3. The options, ``--numLines`` and ``--lineLength``,
+        specify dimensions for creating ``fileToSort.txt`` if it does not already exist. If it exists, this workflow
```

Comment:
>  The options, ``--numLines`` and ``--lineLength``, specify dimensions for creating ``fileToSort.txt`  if it does not already exist. If it exists, this workflow will use the existing file.

This information is repeated in the next paragraph and is redundant. If you don't like that description then fix it up instead of adding this one.

>Delete ``fileToSort.txt`` before moving on to #3.

Why? I think you should explain this a little more. Also I think the `..note::` is not really necessary.

Reply:
Jesse, can you approve this after our conversation last week?